portfolio/src/main/webapp/script.js

```diff
@@ -53,3 +53,10 @@ function deleteTask(task) {
   params.append('id', task.id);
   fetch('/delete-quote', {method: 'POST', body: params});
 }
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
This is a bit confusing - you're creating a map and assigning it to a map variable, but not doing anything with it. What is it there for?